sir_complainsalot/delphi_sir_complainsalot/check_source.py

```diff
@@ -87,26 +88,36 @@ def check_source(data_source, meta, params, grace, logger):
         logger.info("Retrieving signal",
                     source=data_source,
                     signal=row["signal"],
-                    start_day=(row["max_time"] -
-                               gap_window).strftime("%Y-%m-%d"),
-                    end_day=row["max_time"].strftime("%Y-%m-%d"),
+                    start_day=(datetime.now() - timedelta(days = 14)).strftime("%Y-%m-%d"),
+                    end_day=datetime.now().strftime("%Y-%m-%d"),
                     geo_type=row["geo_type"])
 
         latest_data = covidcast.signal(
             data_source, row["signal"],
-            start_day=row["max_time"] - gap_window,
-            end_day=row["max_time"],
+            start_day=datetime.now() - timedelta(days = 14),
+            end_day=datetime.now(),
             geo_type=row["geo_type"]
         )
 
         # convert numpy datetime values to pandas datetimes and then to
         # datetime.date, so we can work with timedeltas after
         unique_dates = [pd.to_datetime(val).date()
```

Comment:
i know this isn't affected in this PR (though the unique issues is), but if this is unique values, should it be wrapped in `set()`?

Reply:
I feel like that only really matters if we were to add more values to it (to make sure that it really stayed a unique set) -- since we are using unique(), the list should still fullfill the contract of the variable name. wydt?

Reply:
oh i didnt even see the unique(), that makes sense to me. is unique() guaranteed to return them in the right order? Based on the gap_days computation line, seems like you want them monotonically increasing.

Reply:
unique() is order-preserving, so if they start out ordered, they'll stay that way

Reply:
:+1: